### PR TITLE
create model and changeset

### PR DESCRIPTION
**Pull request by IhorKalaur**

### CODE REVIEW

**Review by ohatsenko — APPROVED** [review]

File: src/main/java/com/example/bookstore/repository/CartItemRepository.java [hunk 1]
```diff
@@ -1,7 +1,14 @@
...
 import com.example.bookstore.model.CartItem;
+import java.util.Optional;
+import java.util.Set;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
 
 public interface CartItemRepository extends JpaRepository<CartItem, Long> {
+    Optional<CartItem> findCartItemById(Long id);
+
+    @Query(value = "SELECT * FROM cart_item c WHERE c.shopping_cart_id = :id", nativeQuery = true)
```
Comment (vovasalyha) [inline src/main/java/com/example/bookstore/repository/CartItemRepository.java]: is this native query required? You even can remove `@Query` as Spring would parse query from the method name

File: src/main/java/com/example/bookstore/service/impl/OrderServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+@RequiredArgsConstructor
+public class OrderServiceImpl implements OrderService {
+    private final ShoppingCartRepository shoppingCartRepository;
+    private final OrderRepository orderRepository;
+    private final OrderItemRepository orderItemRepository;
+    private final UserService userService;
+    private final OrderMapper orderMapper;
+    private final OrderItemMapper orderItemMapper;
+
+    @Override
```
Comment (vovasalyha) [inline src/main/java/com/example/bookstore/service/impl/OrderServiceImpl.java:41]:
```suggestion
    @Override
    @Transactional
```

File: src/main/java/com/example/bookstore/service/impl/OrderServiceImpl.java [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+                            .map(orderItemMapper::toDto)
+                            .collect(Collectors.toSet());
+                    orderDto.setOrderItems(orderItemDtos);
+
+                    return orderDto;
+                })
+                .toList();
+    }
+
+    @Override
```
Comment (vovasalyha) [inline src/main/java/com/example/bookstore/service/impl/OrderServiceImpl.java:100]:
```suggestion
    @Override
    @Transactional
```

**Review by vovasalyha — APPROVED** [review]
Approving, but please take a look on the comments

**Pull request merged**